Task1/style.css

```diff
@@ -0,0 +1,29 @@
+body{   
+    color:aliceblue;
+    background-image: url('desktop-wallpaper-linear-gradient-green-blue-pale-green-light-thumbnail.jpg');
+    background-size: 100% 450%;
```

Comment:
Try exploring these CSS properties and see how they work in this case:
background-size: auto|length|cover|contain|initial|inherit;

Reply:
I have tried the given properties. Auto, length, initial and inherit repeat the small image multiple times in order to cover the entire page. Cover and contain display the image twice (one above the other) in order to fill the page. 
As I want a uniform coverage of the page (as a single image), I would like to use percentages to determine the size of the image, so that there is no duplication.